src/styles.css

```diff
@@ -3,7 +3,7 @@
   transition: outline-color 0.3s ease-in-out;
 }
 
-.contentful-inspector--active [data-contentful-field-id][data-contentful-entry-id][data-contentful-locale] {
+.contentful-inspector--active [data-contentful-field-id][data-contentful-entry-id] {
```

Comment:
we should remove the locale also from line 1, otherwise the animation should not work